src/Resources/js/views/mealGuest.ts

```diff
@@ -14,14 +16,16 @@ export default class MealGuestView {
 
         if (0 < this.$slotDropDown.length) {
             this.updateSlots();
-            setInterval(this.updateSlots.bind(this), 3000);
         }
 
         if (this.$participationCheckboxes.length > 0) {
             let participationToggleHandler = new ParticipationGuestToggleHandler(this.$participationCheckboxes);
             new ParticipationPreToggleHandler(participationToggleHandler);
-            new ParticipationGuestCountUpdateHandler(this.$participationCheckboxes);
         }
+
+        let messageSubscriber = new MercureSubscriber($('[data-msg-subscribe-url]').data('msgSubscribeUrl'));
```

Comment:
new MercureSubscriber($('[data-msg-subscribe-url]')
   .data('msgSubscribeUrl'))
   .subscribe(['participation-updates'], ParticipationUpdateHandler.updateParticipation)
   .subscribe(['slot-allocation-updates'], SlotAllocationUpdateHandler.handleUpdate);

Reply:
1. "$('[data-msg-subscribe-url]').data('msgSubscribeUrl')" is the parameter to the constructor.
2. Subscribe is not a jquery method.